### PR TITLE
Updated terminology

### PR DESCRIPTION
**Pull request by hannestschofenig**
Based on the discussions at IETF#103 (Bangkok) I have updated the terminology to align it with the terms used by the ITU-T, as described in this presentation https://datatracker.ietf.org/meeting/103/materials/slides-103-suit-liaison-statement-from-itu-t-sg17-00

### CODE REVIEW

**Comment by henkbirkholz** [comment]
I really like this update to the draft!

A few comments (based on diff, pls excuse lack of context, if not included in diff):

"The firmware image can be confidentiality protected": Could this be a good place to differentiate confidentiality wrt data at rest (source) and data in motion (from source)?

"A single device may need to obtain more than one firmware image  and manifest to succesfully perform an update.": Could this be a good place to highlight that basically every device should be assumed to be a composite device (worst case a composite device with only one component). This generic assumption would prevent the assumption that pushing a single "blob" could be enough and encourage to take into account dependencies and the increasing complexity of even the smallest things?

"Status Tracker": Maybe the inclusion of term "campaign" beneficial to this section?

"Firmware Server: Entity [...]": I am not a big fan of the super generic term entity, but maybe that is not of importance here...

"Microcontroller": Maybe highlighting the characteristic of being "independent" and providing its own "computing context" is of relevance here?

"Client-initiated updates": Is the conceptual term of "polling" excluded here on purpose?

"LwM2M Firmware Update object": Is there a reason to get protocol specific in an architecture document at this point (there might be!).

Viele Grüße,

Henk

**Comment by hannestschofenig** [comment]
I added new text and re-arranged things a bit. A few remarks:

"The firmware image can be confidentiality protected": Could this be a good place to differentiate confidentiality wrt data at rest (source) and data in motion (from source)?

[Hannes] I added a forward reference to the architecture section where e2e security is discussed. 

"A single device may need to obtain more than one firmware image and manifest to succesfully perform an update.": Could this be a good place to highlight that basically every device should be assumed to be a composite device (worst case a composite device with only one component). This generic assumption would prevent the assumption that pushing a single "blob" could be enough and encourage to take into account dependencies and the increasing complexity of even the smallest things?

[Hannes] I changed the text and included the definition of a Device where I point out that it has one or multiple MCUs. Let me know if the text is better now. 

"Status Tracker": Maybe the inclusion of term "campaign" beneficial to this section?

[Hannes] How do you think the term could be included here? 

"Firmware Server: Entity [...]": I am not a big fan of the super generic term entity, but maybe that is not of importance here...

[Hannes] OK. I changed the definition. 

"Microcontroller": Maybe highlighting the characteristic of being "independent" and providing its own "computing context" is of relevance here?

[Hannes] Can you make a suggestion? The current text is taken from Wikipedia and stripped down to the parts we care about. 

"Client-initiated updates": Is the conceptual term of "polling" excluded here on purpose?

[Hannes] I added the term. 

"LwM2M Firmware Update object": Is there a reason to get protocol specific in an architecture document at this point (there might be!).

[Hannes] I provided two examples and I believe I need to be more specific in the examples so they actually make sense.

File: draft-ietf-suit-architecture.md [hunk 1]
```diff
@@ -90,20 +90,26 @@ When developing IoT devices, one of the most difficult problems
 to solve is how to update the firmware on the device. Once the 
 device is deployed, firmware updates play a critical part in its 
 lifetime, particularly when devices have a long lifetime, are 
-deployed in remote or inaccessible areas or where manual 
-intervention is cost prohibitive or otherwise difficult. The need 
-for a firmware update may be to fix bugs in software, to add new 
-functionality, or to re-configure the device.
+deployed in remote or inaccessible areas where manual 
+intervention is cost prohibitive or otherwise difficult. Updates 
+to the firmware of an IoT device is done to fix bugs in software,
```
Comment (d3zd3z) [inline draft-ietf-suit-architecture.md]: Updates...are done...

File: draft-ietf-suit-architecture.md [hunk 2]
```diff
@@ -382,46 +385,43 @@ There are three broad classifications of update operating modes.
...
-device that an update is available. This can be accomplished via 
+The notification step consists of the status tracker informing the 
+firmware consumer that an update is available. This can be accomplished via 
 polling (client-initiated), push notifications (server-initiated), 
 or more complex mechanisms.
 
 The pre-authorisation step involves verifying whether the entity 
 signing the manifest is indeed authorized to perform an update. 
-The device must also determine whether it should fetching and 
+The firmware consumer must also determine whether it should fetching and
```
Comment (d3zd3z) [inline draft-ietf-suit-architecture.md]: ...whether it should fetch and process ...

**Review by d3zd3z — CHANGES_REQUESTED** [review]
Overall, good update, just a few grammar errors is all.

**Pull request closed without merging**